packages/clusview/samplers/clusters/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Functions for clustering algorithms such as HDBSCAN
```

Comment:
I would again avoid mentioning specifics here, let's leave this to Intellisense. Also, missing `.` at the end of the sentence.